components/math/multiplicationConnect/GameBoard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+
+const createGrid = () => {
+    let arr = [];
```

Comment:
This should be a state variable inside your component

Reply:
Good point